ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml

```diff
@@ -106,6 +114,25 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-rosa
+  skip_if_only_changed: ^.*(md|adoc)$|^LICENSE$
+  steps:
+    cluster_profile: quay-aws
+    env:
+      BYO_OIDC: "true"
+      CHANNEL_GROUP: stable
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: 4.15.8
+      REGION: us-east-2
+    test:
+    - as: test-ocp4-pci-dss-node
+      cli: latest
+      commands: make e2e-roas
```

Comment:
seems a typo issue:
`      commands: make e2e-rosa`?

Reply:
Good question - I'm implementing that in https://github.com/ComplianceAsCode/compliance-operator/pull/515